include/mxnet/c_api.h

```diff
@@ -663,6 +663,7 @@ MXNET_DLL int MXNDArrayReshape(NDArrayHandle handle,
 MXNET_DLL int MXNDArrayReshape64(NDArrayHandle handle,
```

Comment:
Did you check that this does not break our other frontend APIs? In this PR, you only adapter the Python version.